addons/website/models/theme_models.py

```diff
@@ -222,13 +222,6 @@ class Theme(models.AbstractModel):
         'website.template_header_hamburger',
         'website.template_header_vertical',
         'website.template_header_sidebar',
-        'website.template_header_slogan',
-        'website.template_header_contact',
-        'website.template_header_boxed',
-        'website.template_header_centered_logo',
-        'website.template_header_image',
-        'website.template_header_hamburger_full',
-        'website.template_header_magazine',
```

Comment:
It's good that you split your work into multiple commits but ideally they should make sense on their own. In this first commit, you seem to remove template_header_default, but you keep it mentioned here in theme_models. You also remove it and its alignment options, making all the editor widgets empty (which does not make sense if I checkout this commit on its own)... but in a later commit you reintroduce it with the same xml id and with the same options for alignments. Could not you remove only what will be really removed in this commit and update what will be update into the related commit. This is important for review but also for the history. If I later stumble upon the "default header" commit, I cannot see what changed compared to the previous one... I need to find another commit for that :grimacing:

Reply:
Hello @qsm-odoo ! 👋 

Indeed, I understand now this history issue 👍 
We've updated the commits structure and rebased the branch to fix this.
Let me know if this is ok for you :)